src/Build/Graph/GraphBuilder.cs

```diff
@@ -499,18 +502,44 @@ private ParsedProject ParseProject(ConfigurationMetadata configurationMetadata)
         {
             // TODO: ProjectInstance just converts the dictionary back to a PropertyDictionary, so find a way to directly provide it.
             var globalProperties = configurationMetadata.GlobalProperties.ToDictionary();
+            ProjectGraphNode graphNode;
+            ProjectInstance projectInstance;
+            var DynamiclySetPlatform = IsDynamicPlatformNegotiationEnabaled && !configurationMetadata.IsSetPlatformHardCoded;
```

Comment:
Typo.

Changing to bool is a nit, but it matches other vars here so it's fine (I still personally don't like it but eh, it's been that way)

```suggestion
            bool NegotiatePlatform = IsDynamicPlatformNegotiationEnabaled && !configurationMetadata.IsSetPlatformHardCoded;
```

Reply:
What typo are you referring to here

Reply:
Another nit: it shouldn't be capitalized

Reply:
Fixed variable name